examples/with-apollo/lib/apollo.js

```diff
@@ -56,7 +56,8 @@ export function withApollo (PageComponent, { ssr = true } = {}) {
 
         try {
           // Run all GraphQL queries
-          await require('@apollo/react-ssr').getDataFromTree(
+          const { getDataFromTree } = await import('@apollo/react-ssr')
```

Comment:
Should this be the following?
```suggestion
          const { getDataFromTree } = await import('@apollo/react-ssr').then(e => e.default)
```

Reply:
Afaik they use names exports: https://github.com/apollographql/react-apollo/blob/master/packages/ssr/src/index.ts#L1

I am testing every change I do in production and development mode. But let me take a deeper look. I have never used `async imports` in node context. Might be interesting what this compiles to.

Reply:
A bit off topic 🤔

#### Using `async import(...)`

```js
const {
  getDataFromTree
} = await Promise.resolve(/* import() */).then(__webpack_require__.t.bind(null, "7fVs", 7));
```

#### Using `require(...)`

```js
const {
  getDataFromTree
} = __webpack_require__("7fVs");
```

Reply:
Why not just import this at the top of the file? What is the purpose of making it dynamic?

Reply:
This module will be shipped to the client if you import it at the top of the file, even though it is only needed on the server.

Keep in mind that the code of your next application will run in multiple different contexts (node.js & Browser).

Reply:
Interesting, that makes sense now. Thank you for clarifying!